pkgs/development/ocaml-modules/reason-native/default.nix

```diff
@@ -0,0 +1,43 @@
+{ newScope, lib, fetchFromGitHub, callPackage, buildDunePackage, atdgen, junit, qcheck-core, re, reason, reason-native }:
+
+let
+  generic = (somePath:
+    let
+      prepkg = import somePath {
```

Comment:
I guess that this should use `callPackage` from the scope instead of `import`.

Reply:
`callPackage` does not work here. At least the way it is. Because generic is called on an attrs, not a package.

Reply:
I fear that overriding the scope may not work properly…